pkgs/development/interpreters/python/cpython/3.6/default.nix

```diff
@@ -64,6 +64,14 @@ in stdenv.mkDerivation {
     substituteInPlace configure --replace '-Wl,-stack_size,1000000' ' '
   '';
 
+  patches = optionals (!(stdenv.cc.isGNU or false)) [
```

Comment:
@LnL7 has a valid point but I would prefer to keep the Python interpreter as vanilla as possible and therefore only apply patch when really necessary.